tsconfig.json

```diff
@@ -51,6 +51,8 @@
     /* Experimental Options */
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+    "incremental": true, /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
```

Comment:
small improvement for local dev, `tsc` runs incremental build to compile faster.